.gitignore

```diff
@@ -58,3 +58,6 @@ tba-unit-tests/FailureDiffs
 # Ignore some Metadata from Fastlane with sensitive information
 fastlane/metadata/trade_representative_contact_information
 fastlane/metadata/review_information
+
+# rbenv
+.ruby-version
```

Comment:
What's this about?

Reply:
It's used by `rbenv`, a `ruby` virtual environment manager; like `python`'s `virtualenv`.  The `.ruby-version` file specifically tells `rbenv` what version of `ruby` to use for the given folder.
It's essentially a required thing for `ruby` devs.  The normal thing to do is just ignore the file, but if that's an issue I can take it out.

Reply:
Gotcha - if we sync about this, we could update the setup docs to make sure everyone is using the same version of Ruby. That seems ideal.